packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -942,3 +967,27 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
     description.add('viewport: ${_viewportDimension?.toStringAsFixed(1)}');
   }
 }
+
+/// A notification that a [Scrollable] widget's contents metrics has changed.
```

Comment:
Similar updates to the docs here. Also, discuss how this is different from the other ScrollNotifications.